.github/workflows/abi.yaml

```diff
@@ -68,7 +68,7 @@ jobs:
           cp build_abi/install_lib/* `pg_config --pkglibdir`
           chown -R postgres /mnt
           set -o pipefail
-          sudo -u postgres make -C build_abi -k regresscheck regresscheck-t regresscheck-shared | tee installcheck.log
+          sudo -u postgres make -C build_abi -k regresscheck regresscheck-t regresscheck-shared IGNORES="memoize" | tee installcheck.log
```

Comment:
Pretty sure memoize shouldn't fail anymore on 14.2 and other recent releases unless we test against PG <= 14.1. Maybe this change is not needed.

Reply:
Hmm... OK then 🤷‍♂️